pkgs/development/python-modules/sensoterra/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  requests,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "sensoterra";
+  version = "2.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-WfjTOns5OPU8+ufDeFdDGjURhBWUFfw/qRSHQazBL04=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ requests ];
+
+  # Module has no tests
+  doCheck = false;
```

Comment:
https://gitlab.com/sensoterra/public/python/-/blob/main/tests/test_customerapi.py?ref_type=heads

has one test, not sure if is usable w/o network